### PR TITLE
feat(sbb-stepper): add size 's'

### LINKED ISSUE #2765: Stepper: Size S

### PR DESCRIPTION
**Pull request by TomMenga**
This PR Closes #2765

### CODE REVIEW

**Review by DavideMininni-Fincons — APPROVED** [review]
I would add some test for the size change

File: src/elements/stepper/stepper/readme.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ Use an `aria-label` attribute to describe the purpose of the stepper. The `sbb-s
 | `orientation`    | `orientation`     | public  | `SbbOrientation`                 | `'horizontal'` | Steps orientation, either horizontal or vertical.                                 |
 | `selected`       | -                 | public  | `SbbStepElement \| undefined`    |                | The currently selected step.                                                      |
 | `selectedIndex`  | `selected-index`  | public  | `number \| undefined`            |                | The currently selected step index.                                                |
+| `size`           | `size`            | public  | `'s' \| 'm'`                     | `'m'`          | Size variant, either l or m.                                                      |
```
Comment (jeripeierSBB) [inline src/elements/stepper/stepper/readme.md:113]: Description doesn't match possible values

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
LGTM, thank you! one docs change needed

**Comment by TomMenga** [comment]
> I would add some test for the size change

Makes sense, I'll add them

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Comment by mcilurzo** [comment]
UX Review: 

https://www.figma.com/design/9r6xSfNmEfCFxl1yFYedrj?node-id=43627-137922#927954441
https://www.figma.com/design/9r6xSfNmEfCFxl1yFYedrj?node-id=43627-137922#927952423

**Comment by mcilurzo** [comment]
LGTM. Thank you!

**Pull request merged**